.github/workflows/multi-platform.yml

```diff
@@ -0,0 +1,160 @@
+name: Docker Publish Multi-Platform
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+    # tags:
+    #   - 'v*'
+
+env:
+  REGISTRY: docker.io
+  # REGISTRY: ghcr.io # For GitHub Container Registry
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    name: Build and Push Multi-Platform Docker Image
+    runs-on: ubuntu-latest
```

Comment:
We could consider running the ARM builds on a natively ARM worker now since Github supports this in public preview: https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/ -- should theoretically speed things up significantly cc: @jasongdove

Reply:
Added an arm runner for arm builds